metasploit-framework.gemspec

```diff
@@ -216,7 +216,7 @@ Gem::Specification.new do |spec|
   # SSH server library with ed25519
   spec.add_runtime_dependency 'hrr_rb_ssh-ed25519'
   # Needed for irb internal command
-  spec.add_runtime_dependency 'irb'
+  spec.add_runtime_dependency 'irb', '~> 1.7.4'
```

Comment:
irb 1.8 pulls in psych which would cause issues right now; should be able to unpin this once we're officially on Ruby 3.1